client/components/deposits-status/index.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import { Icon } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+
+const DepositsEnabled = ( props ) => {
+	const { depositsEnabled, iconSize } = props;
+	let className = 'account-details__info__green';
```

Comment:
(minor) [modifiers in BEM](http://getbem.com/naming/) are by convention usually prefixed by `--` (double dash) - so in this case `green` and `yellow` should be prefixed by `--` instead of `__`